### ISSUE TITLE
see if display view is enabled in group list/tiles

### ISSUE THREAD

**Issue by evaklopp** [buzzn/scrum#99]
**User story**

As a (kind of user)
I want (a goal)
so that (a reason).

---

**Definition of done**

- [ ] ...

---

**Technical details**

_Frontend_

- [ ] ...

_Backend_

- [ ] ...

**Comment by eharit** [comment]
Suggest we'd use:
<img width="910" alt="screen shot 2018-01-03 at 16 27 17" src="https://user-images.githubusercontent.com/6729674/34526533-0337e756-f0a3-11e7-9164-72c5e91d07ec.png">

**Comment by eharit** [comment]
Currently the *list* of Energy Groups are sortable by 'Groupname' only. Shall we make it sortable by Display View, too? It would make sense if PhO needs to find quickly all buildings that do not display it for some reason.

**Comment by dottorer** [comment]
Not crucial but can't hurt if it doesn't add visual clutter to the header.

**Comment by eharit** [comment]
Yep, it's just a pair of carets next to the list header label, means no clutter at all, see mockup above

**Comment by evaklopp** [comment]
@eharit see comments in InVision
